2022_Recipe1

```diff
@@ -0,0 +1,37 @@
+File: Recipe 1 - Home Made Unique Crispy Chicken Recipe
+Type: Documentation
+Author: upxksha
+
+Follow the steps accordingly for the most satisfying results.
+
+Ingredients List 1: 
+Chicken 1kg
+Vinegar
+Onion
+Black Pepper
+Chilli Powder
+Chilli Flakes
+Tomato Sauce
+
+Ingredients List 2:
+Flour
+Corn Flour
+Black Pepper
+Onion
+Chilli Powder
+Chilli Flakes
+
+Step 1: 
+Mix the Ingredients List 1 well and let it marinate for about 1-2 hours.
+
+Step 2: 
+Then mix the Ingredients List 2 very well meanwhile.
+
+Step 3: 
+Take out some ice cubes and let it melt in a glass. 
+
+Step 4: 
+Add the ingredients list 2 all over the chicken properly. 
+Then put the chicked in the iced water slowly.
```

Comment:
```suggestion
Then put the chicken in the iced water slowly.
```

On line 31 it says `Take out some ice cubes and let it melt in a glass.`

This implies that I am to fit a whole chicken into a glass of ice water. I tried this and received a `ChickenNoFitException`.